### DISCUSSION TITLE
FNF Links

### DISCUSSION THREAD

**Discussion by Bigfoot9999** [BigfootsGS/BigfootsGS.github.io#15] [General]
https://docs.google.com/document/d/1Ha0VgSXvb4bClUJYzDocY83hlYhINFZzUnxS09c7kzs/edit

**Comment by hhmsgamesmcps** [comment]
It says access denied

**Reply by Bigfoot9999** [reply]
try this https://docs.google.com/document/d/1Ha0VgSXvb4bClUJYzDocY83hlYhINFZzUnxS09c7kzs/edit?usp=sharing
